src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -63,9 +63,16 @@ public class HubEndPoint<THub, TClient> : EndPoint where THub : Hub<TClient>
 
                 using (var scope = _serviceScopeFactory.CreateScope())
                 {
-                    var hub = scope.ServiceProvider.GetService<THub>() ?? Activator.CreateInstance<THub>();
-                    InitializeHub(connection, hub);
+                    bool created;
```

Comment:
`var created = false`? 

Also, I'd like someone to take a look at the name `created` in this context (and in `CreateHub` method below). It's set to true only if it's created by `Activator`, though even if it's coming through a `GetService` call, someone have "created" it before :)